<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionForm</class>
 <widget class="QDialog" name="QuestionForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>479</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QWidget" name="BackGround1" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#BackGround1 {
	background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(49, 108, 117, 255), stop:0.5 rgba(64, 140, 153, 255), stop:1 rgba(49, 108, 117, 255));
	border-style: solid;
	border-width: 3px;
	border-color: rgb(24, 54, 58);
	border-radius: 30px;
	font-family: &quot;Arial&quot;;
	font-size: 20px;
	font-weight: bold;
	color: #fff;
}</string>
     </property>
     <layout class="QGridLayout" name="BackGround">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item row="3" column="0">
       <widget class="QPushButton" name="panoButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
     background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(89, 155, 179, 255), stop:0.2 rgba(64, 140, 153, 255));
	border-style: solid;
	border-width: 1px;
	border-color: rgb(24, 54, 58);
	border-radius: 10px;
	font-family: &quot;Arial&quot;;
	font-size: 20px;
	font-weight: bold;
	color: #fff;
 }
 QPushButton:hover {
     background: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0.011, stop:0 rgba(89, 155, 179, 255), stop:0.2 rgba(64, 140, 153, 255));
 }
 QPushButton:pressed {
     background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(49, 108, 117, 255), stop:0.2 rgba(64, 140, 153, 255));
 }</string>
        </property>
        <property name="text">
         <string>П О Д И В И Т И С Я  П А Н О Р А М У</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>26</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#fff</string>
        </property>
        <property name="text">
         <string>З А П И Т А Н Н Я  1 / 11</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="GridLayout" native="true">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border : 0px solid blue;
border-top : 3px solid rgb(24, 54, 58);
background-color: rgb(24, 54, 58);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="4" column="0">
          <widget class="QRadioButton" name="answ2">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff</string>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="questionLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff</string>
           </property>
           <property name="text">
            <string>1. Як у цій ситуації виконати поворот праворуч?</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="answ1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff</string>
           </property>
           <property name="text">
            <string>Миготіння зеленого сигналу попереджає про несправність світлофора,
тому слід керуватися вимогами знаків пріоритету.</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="answ3">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff</string>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QRadioButton" name="answ4">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff</string>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="answButton">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
     background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(89, 155, 179, 255), stop:0.2 rgba(64, 140, 153, 255));
	border-style: solid;
	border-width: 1px;
	border-color: rgb(24, 54, 58);
	border-radius: 10px;
	font-family: &quot;Arial&quot;;
	font-size: 20px;
	font-weight: bold;
	color: #fff;
 }
 QPushButton:hover {
     background: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0.011, stop:0 rgba(89, 155, 179, 255), stop:0.2 rgba(64, 140, 153, 255));
 }
 QPushButton:pressed {
     background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(49, 108, 117, 255), stop:0.2 rgba(64, 140, 153, 255));
 }</string>
        </property>
        <property name="text">
         <string>В І Д П О В І С Т И</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
